dir.traversal.targets

```diff
@@ -172,6 +172,29 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+  <Target Name="GenerateTestProjectJsons">
```

Comment:
Well actually now I see that you already used the name GenerateAllTestProjectJsons in the build.proj, perhaps we could rename that one and have this one with the "All" it its name.